run_tests.py

```diff
@@ -1,4 +1,6 @@
-from tests.test_azure_storage_api import TestMountContainerFunction, TestGetBlob
```

Comment:
why this manual set of tests?

unittest are automatically discovered using this command:

```
python -m unittest discover -s tests
```

does that not work?

Reply:
isn't the tests run/debug facility built-in Visual Studio Code sufficient for that purpose?

https://code.visualstudio.com/docs/python/testing#_debug-tests

Reply:
It works for testing, but the debug functionality loops indefinitely. Also, even if we are using unittest, if pytest is not installed, the Visual Studio Code module for the test won't discover the test as it will crash trying to import pytest.

Reply:
The issue can be resolved by using `Developer: Reload Window`

[vscode-python issue22845](https://github.com/microsoft/vscode-python/issues/22845#issuecomment-1954570690)

The details issue:

[vscode-python issue21688](https://github.com/microsoft/vscode-python/issues/21688#issuecomment-1949213713)

it seems to be a recent issue since the posts are from last month

Reply:
Seems like the issue is fixed, opened a new Codespaces, and the test charged without any trouble.